### PR TITLE
Add a test template for ledger

### PR DESCRIPTION
**Pull request by IsaacG**

### CODE REVIEW

File: exercises/practice/ledger/.meta/template.j2 [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        ]
+        expected = '\n'.join([
+            {%- for line in case["expected"] %}
+            '{{- line -}}',
+            {%- endfor %}
+        ])
+        self.assertEqual(format_entries(currency, locale, entries), expected)
+    {%- endfor %}
+
+{{ macros.footer() }}
```
Comment (BethanyG) [inline exercises/practice/ledger/.meta/template.j2:24]:
```suggestion
```

Comment (BethanyG) [reply exercises/practice/ledger/.meta/template.j2:24]: We're removing these as a first step toward (eventually) changing over to Pytest syntax.

**Comment by BethanyG** [comment]
@IsaacG -- since this hasn't been resolved in `problem-specs`, I am closing this for now.  We can revisit/reopen later.

**Comment by IsaacG** [comment]
@BethanyG The canonical data has merged. Should I file a new PR?

**Comment by BethanyG** [comment]
No.  I think I want to hold off on any additional templates at this time.  Thanks tho!

**Pull request closed without merging**